webapp/views/partials/users.jade

```diff
@@ -2,7 +2,7 @@ extends ../layout
 
 block content
```

Comment:
hmm, this shouldn't be here in a partials file. Can you fix that too? I presume this also has "extends ../layout" which should also be removed.